install.sh

```diff
@@ -271,21 +283,21 @@ node(){
 compile_tmux(){
   cd "$C9_DIR"
   echo ":Compiling libevent..."
-  tar xzf libevent-2.0.21-stable.tar.gz
-  rm libevent-2.0.21-stable.tar.gz
-  cd libevent-2.0.21-stable
+  tar xzf libevent-2.0.22-stable.tar.gz
+  rm libevent-2.0.22-stable.tar.gz
+  cd libevent-2.0.22-stable
   echo ":Configuring Libevent"
-  ./configure --prefix="$C9_DIR/local"
+  ./configure --disable-shared --prefix="$C9_DIR/local"
```

Comment:
I don't know what this change is for, but I trust you know what you're doing

Reply:
tmux changes were done by @mutahhir, and have been tested on v4